packages/@angular/cli/commands/build.ts

```diff
@@ -125,6 +126,13 @@ export const baseBuildCommandOptions: any = [
     default: true,
     aliases: ['dop'],
     description: 'Delete output path before build.'
+  },
+  {
+    name: 'extract-licenses',
+    type: Boolean,
+    default: true,
```

Comment:
Could you specify in the description that this flag is only used in production builds (when `--target=prod`)? Seems like people might think it will affect dev builds as well.